core/tests/karma.conf.ts

```diff
@@ -105,8 +106,12 @@ module.exports = function(config) {
     singleRun: true,
     customLaunchers: {
       Chrome_Travis: {
-        base: 'Chrome',
-        flags: ['--no-sandbox']
+        base: isDocker ? 'ChromeHeadless' : 'Chrome',
```

Comment:
Add comments above this and the next line explaining the rationale.

Reply:
Done.